airflow/contrib/operators/bigquery_operator.py

```diff
@@ -64,8 +64,9 @@ class BigQueryOperator(BaseOperator):
     :param udf_config: The User Defined Function configuration for the query.
         See https://cloud.google.com/bigquery/user-defined-functions for details.
     :type udf_config: list
-    :param use_legacy_sql: Whether to use legacy SQL (true) or standard SQL (false).
-    :type use_legacy_sql: boolean
+    :param use_legacy_sql: Whether to use legacy SQL (true) or
+    standard SQL (false).
```

Comment:
Can you give a tab here if you want to break `standard SQL (false).` to a new line?